src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -144,9 +144,17 @@
 
     <!-- If VersionsFiles were passed on the command-line, convert them to an item group -->
     <ItemGroup>
-      <_VersionsFiles Include="$(VersionsFiles)" />
+      <_VersionsFiles Include="$(VersionsFiles)" />    
+      <SupportsDefinitionFileGroup Condition="'$(SupportsDefinitionFile)' == ''" Include="$(MSBuildThisFileDirectory)../packages/Microsoft.DotNet.BuildTools.TestSuite/**/runtime.json" />
```

Comment:
Whatever we use I want it to be configurable by repos so we don't have to update buildtools to change it.  I'd also like to minimize the boilerplate for configuring it.